lxd/instance/drivers/driver_lxc.go

```diff
@@ -739,6 +739,13 @@ func (d *lxc) initLXC(config bool) error {
 		}
 	}
 
+	if d.state.OS.CoreScheduling {
+		err = lxcSetConfigItem(cc, "lxc.sched.core", "1")
```

Comment:
In the current implementation we always make use of core scheduling. Do we want to be able to configure this through a config option?

Reply:
*lxd config option

Reply:
For now, just always doing it seems fine to me I think.

Reply:
sounds good